test/hotspot/jtreg/runtime/cds/MaxMetaspaceSize.java

```diff
@@ -44,9 +44,8 @@ public static void main(String[] args) throws Exception {
     processArgs.add("-Xshare:dump");
 
     if (Platform.is64bit()) {
-      processArgs.add("-XX:MaxMetaspaceSize=3m");
+      processArgs.add("-XX:MaxMetaspaceSize=8m");
```

Comment:
On the contrary. Minimum metaspace usage to run a simple HelloWorld went way down. 

Before: 10,00 MB reserved,       4,75 MB ( 48%) committed 
Now: 12,00 MB reserved,     448,00 KB (  3%) committed.

mainly because now we commit the large chunk for the boot loader only on demand. Reservation (vsize) numbers are somwhat chunkier now since the default size for a VirtualSpaceNode went up from 4 to 8MB.

You now can get away with -XX:MaxMetaspaceSize=~500K to start a simple program.

While writing this, I wondered why I made this change to your test. It is not needed for the current version, probably a remnant from some earlier prototype. I will revert this line and check the other tests too.